client/analytics/settings/index.scss

```
@@ -11,6 +11,8 @@
 		margin-left: 15%;
 	}
 
+		margin-bottom: $gap-largest;
```

Thank you for applying the changes @ronakganatra9!

Could you just remove the extra tab here?

```suggestion
	margin-bottom: $gap-largest;
```

And one more suggestion, what do you think about moving it above the previous block so it looks like this?
```SCSS
.woocommerce-settings__actions {
	margin-bottom: $gap-largest;

	@include breakpoint( '>1280px' ) {
		margin-left: 15%;
	}
```
This way we have the generic declarations on top and the media queries below.

Once that's done, I will proceed merging.

---

Sounds Good, Applied. Thanks.